### DISCUSSION TITLE
Dual LG Ultrafine 27" Primary Monitor Only Adjusted

### DISCUSSION THREAD

**Discussion by ontoarch** [MonitorControl/MonitorControl#1413] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

When using dual LG ultrafine 27" monitors, the controls, specifically brightness only works on the primary monitor and adjusting the secondary (extended display) adjusts the primary.    The hardware is M2 Max Pro MBP 2023 May build running Ventura 13.5.

### Steps to reproduce

Upon startup using dropdown adjust monitor brightness

### Expected behavior

To control both displays independently

### Environment Information (please complete the following information)

```markdown
- macOS version:13.5
- Mac model:M2 Max Pro MBP
- MonitorControl version:4.1.0
- Monitor(s):LG Ultrafine 27"
- Apple Silicon/M1 (yes or no): yes

Apple M2 Max:

  Chipset Model:	Apple M2 Max
  Type:	GPU
  Bus:	Built-In
  Total Number of Cores:	38
  Vendor:	Apple (0x106b)
  Metal Support:	Metal 3
  Displays:
LG UltraFine:
  Resolution:	5120x2880 (5K/UHD+ - Ultra High Definition Plus)
  UI Looks like:	2560 x 1440 @ 60.00Hz
  Main Display:	Yes
  Mirror:	Off
  Online:	Yes
  Rotation:	Supported
  Automatically Adjust Brightness:	No
LG UltraFine:
  Resolution:	5120x2880 (5K/UHD+ - Ultra High Definition Plus)
  UI Looks like:	2560 x 1440 @ 60.00Hz
  Mirror:	Off
  Online:	Yes
  Rotation:	Supported
  Automatically Adjust Brightness:	No
```

**Comment by waydabber** [comment]
This issue has complex reasons with multi-monitor setups and addressing them in MonitorControl would require some deeper changes. You can try the latest 2.x version of BetterDisplay to see if you have this issue: https://github.com/waydabber/BetterDisplay/releases
